### DISCUSSION TITLE
App shows deleted discussions in list, then “Something went wrong.”

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#3273] [Mobile] [answered]
When I view this feedback discussion forum in the GitHub mobile app, it lists the following titles among others:

* developer prograg Around#1
* SECURITY
* javascript:void(0);

However, when I try to open any of those three, the app just shows: "Something went wrong. Try Again"

The web site does not list those three discussions at all, when I search for words in their titles. So, I think the discussions have been deleted but they still appear in the mobile app for some reason.

Strangely, when the mobile app lists any of those discussions, it does not show the screen name of the user who started the discussion, even though it does show "started a discussion" preceded by an image that looks like a profile picture.

**Comment by candyho** [comment, marked as answer]
Hey thanks for reporting this bug. I have created an issue in our internal repo to track this bug and we will work on a fix soon!
